homeassistant/components/aemet/sensor.py

```diff
@@ -51,172 +83,262 @@
-        key=ATTR_API_FORECAST_PRECIPITATION_PROBABILITY,
-        name="Precipitation probability",
+    AemetSensorEntityDescription(
+        key=f"forecast-daily-{ATTR_API_FORECAST_PRECIPITATION_PROBABILITY}",
+        keys=[
+            AOD_TOWN,
+            AOD_FORECAST_DAILY,
+            AOD_FORECAST_CURRENT,
+            AOD_PRECIPITATION_PROBABILITY,
+        ],
+        name="Daily Forecast Precipitation probability",
         native_unit_of_measurement=PERCENTAGE,
     ),
-    SensorEntityDescription(
-        key=ATTR_API_FORECAST_TEMP,
-        name="Temperature",
+    AemetSensorEntityDescription(
+        entity_registry_enabled_default=False,
+        key=f"forecast-hourly-{ATTR_API_FORECAST_PRECIPITATION_PROBABILITY}",
+        keys=[
+            AOD_TOWN,
+            AOD_FORECAST_HOURLY,
+            AOD_FORECAST_CURRENT,
+            AOD_PRECIPITATION_PROBABILITY,
+        ],
+        name="Hourly Forecast Precipitation probability",
+        native_unit_of_measurement=PERCENTAGE,
+    ),
+    AemetSensorEntityDescription(
+        key=f"forecast-daily-{ATTR_API_FORECAST_TEMP}",
+        keys=[AOD_TOWN, AOD_FORECAST_DAILY, AOD_FORECAST_CURRENT, AOD_TEMP_MAX],
+        name="Daily Forecast Temperature",
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         device_class=SensorDeviceClass.TEMPERATURE,
     ),
-    SensorEntityDescription(
-        key=ATTR_API_FORECAST_TEMP_LOW,
-        name="Temperature Low",
+    AemetSensorEntityDescription(
+        key=f"forecast-daily-{ATTR_API_FORECAST_TEMP_LOW}",
+        keys=[AOD_TOWN, AOD_FORECAST_DAILY, AOD_FORECAST_CURRENT, AOD_TEMP_MIN],
+        name="Daily Forecast Temperature Low",
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         device_class=SensorDeviceClass.TEMPERATURE,
     ),
-    SensorEntityDescription(
-        key=ATTR_API_FORECAST_TIME,
-        name="Time",
+    AemetSensorEntityDescription(
+        entity_registry_enabled_default=False,
+        key=f"forecast-hourly-{ATTR_API_FORECAST_TEMP}",
+        keys=[AOD_TOWN, AOD_FORECAST_HOURLY, AOD_FORECAST_CURRENT, AOD_TEMP],
+        name="Hourly Forecast Temperature",
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+    ),
+    AemetSensorEntityDescription(
+        key=f"forecast-daily-{ATTR_API_FORECAST_TIME}",
+        keys=[AOD_TOWN, AOD_FORECAST_DAILY, AOD_FORECAST_CURRENT, AOD_TIMESTAMP],
+        name="Daily Forecast Time",
         device_class=SensorDeviceClass.TIMESTAMP,
     ),
-    SensorEntityDescription(
-        key=ATTR_API_FORECAST_WIND_BEARING,
-        name="Wind bearing",
+    AemetSensorEntityDescription(
+        entity_registry_enabled_default=False,
+        key=f"forecast-hourly-{ATTR_API_FORECAST_TIME}",
+        keys=[AOD_TOWN, AOD_FORECAST_HOURLY, AOD_FORECAST_CURRENT, AOD_TIMESTAMP],
+        name="Hourly Forecast Time",
+        device_class=SensorDeviceClass.TIMESTAMP,
```

Comment:
```suggestion
        device_class=SensorDeviceClass.TIMESTAMP,
        value_fn=lambda value: dt_util.parse_datetime(value)
```

Please adopt other places too

Reply:
Done in f064f9e010966a5c9832823cc324e04c7e348d2a